src/helics/application_api/Federate.hpp

```diff
@@ -335,9 +339,13 @@ class HELICS_CXX_EXPORT Federate {
     specific name of a federate, core, or broker
     @param queryStr a string with the query see other documentation for specific properties to
     query, can be defined by the federate
+    @param mode asynchronous(default) means the query goes on priority channels, synchronous is
+    slower but has more ordering guarantees
```

Comment:
@param mode fast (asynchronous; default) means the query goes on priority channels, ordered (synchronous) is
    slower but has more ordering guarantees